docs/content/about.md

```diff
@@ -31,7 +31,7 @@ The following table shows the version mapping between Flink<sup>®</sup> CDC Con
 | <font color="DarkCyan">2.0.*</font> | <font color="MediumVioletRed">1.13.*</font> |
 | <font color="DarkCyan">2.1.*</font> | <font color="MediumVioletRed">1.13.*</font> |
 | <font color="DarkCyan">2.2.*</font> | <font color="MediumVioletRed">1.13.\*</font>, <font color="MediumVioletRed">1.14.\*</font> |
-| <font color="DarkCyan">2.3.*</font> | <font color="MediumVioletRed">1.13.\*</font>, <font color="MediumVioletRed">1.14.\*</font>, <font color="MediumVioletRed">1.15.\*</font> |
+| <font color="DarkCyan">2.3.*</font> | <font color="MediumVioletRed">1.13.\*</font>, <font color="MediumVioletRed">1.14.\*</font>, <font color="MediumVioletRed">1.15.\*</font>, <font color="MediumVioletRed">1.16.0</font> |
```

Comment:
The guava we should use `30.1.1-jre-16.0` in pom.xml,  and could you create  issue for the failed tests?

Reply:
Fixed. I will retry the tests later. It seems no problem locally.